articles/libraries/lock/v11/index.md

```diff
@@ -0,0 +1,142 @@
+---
+section: libraries
+toc: true
+title: Lock 11 for Web
+description: A widget that provides a frictionless login and signup experience for your web apps.
+img: media/articles/libraries/lock-web.png
+---
+# Lock 11 for Web
+
+Lock is an embeddable login form, [configurable to your needs](/libraries/lock/v11/configuration) and ready for use on web apps. It enables you to easily add social identity providers to Lock, allowing your users to login seamlessly using any provider they want.
+
+::: panel Hosted Login Page
+Lock 11 is a version of Lock tailored for embedded login in your applications. If you use Auth0's [Hosted Login Page](/hosted-pages/login), Lock 11 is not necessary and does not work inside the Hosted Login Page. The Hosted Login Page remains the simplest and most secure method by which to authenticate users for your applications. Lock 11 is designed to make cross-origin authentication work smoothly for when you cannot use the Hosted Login Page and need to embed a login in your application.
```

Comment:
I'm not sure about the "Lock 11 is not necessary" part, it's a little confusing.
How about "If you use Auth0's [Hosted Login Page](/hosted-pages/login), do not upgrade to Lock 11. Instead, keep using the latest version of Lock 10."
This might event warrant some graphic.
Want Lock or Auth0.js embedded in your app? -> Use Lock 11 or Auth0.js 9
Want Lock or Auth0.js in the hosted login page (a.k.a. centralized login) -> Use Lock v10 or Auth0.js 8